.github/workflows/tests.yml

```diff
@@ -36,10 +36,10 @@ jobs:
         run: build_scripts/run_shellcheck.sh
 
       - name: Run flake8
-        run: flake8
+        run: flake8 
 
-      - name: Run pytest (excluding integration tests)
-        run: pytest --verbose -m "not integration" --timer-top-n 10
+      - name: Run pytest Run All Other Tests (excluding integration and e2e)
```

Comment:
This name seems redundant. How about just 
Run pytest excluding integration, e2e and smoke tests